pkg/kubelet/cadvisor/cadvisor_linux_test.go

```diff
@@ -0,0 +1,65 @@
+// +build cgo,linux
+
+/*
+Copyright 2015 The Kubernetes Authors.
```

Comment:
Fixed.

On Mon, Aug 22, 2016 at 7:43 PM Tim St. Clair <notifications@github.com>
wrote:

> In pkg/kubelet/cadvisor/cadvisor_linux_test.go
> <https://github.com/kubernetes/kubernetes/pull/31064#discussion_r75780059>
> :
>
> > @@ -0,0 +1,65 @@
> > +// +build cgo,linux
> > +
> > +/*
> > +Copyright 2015 The Kubernetes Authors.
>
> nit: 2016
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/31064/files/17d69847b7c37244cf0ce33f430d21c6afd6ce80#r75780059>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAANaFk7sxBo5-3AmxuS-gAODN_vYFN7ks5qijQggaJpZM4JpLXD>
> .
>